ethstats/ethstats.go

```diff
@@ -56,23 +56,15 @@ const (
 	chainHeadChanSize = 10
 )
 
-type txPool interface {
-	// SubscribeNewTxsEvent should return an event subscription of
-	// NewTxsEvent and send events to the given channel.
-	SubscribeNewTxsEvent(chan<- core.NewTxsEvent) event.Subscription
-}
-
-type blockChain interface {
-	SubscribeChainHeadEvent(ch chan<- core.ChainHeadEvent) event.Subscription
-}
+// Backend encompasses the backend behaviors needed for the ethstats service.
+type Backend ethapi.Backend
```

Comment:
* no need for defining Backend = ethapi.Backend

* or list out the methods that are necessary here under Backend (what's needed with ethstats), this saves the import and keeps it more independent